### ISSUE TITLE
courses in other languages

### ISSUE THREAD

**Issue by shawna-slh** [w3c/wai-course-list#347]
Shall we encourage courses in other languages to put the title, description, topics, etc. in:
1. both the language of the course and English? **_or_**
2. only the language of the course? **_or_**
3. only English?

_Note that we already deemed translating the course list itself is out of scope._

**Comment by shawna-slh** [comment]
/me leans towards #1, with the lang of the course first and English in parentheses.

However, that could get long (and there's a word-count limit on the description). So maybe just the course title in both languages and the rest in the language of the course?

**Comment by carlosapaduarte** [comment]
We already have one course in the list following option 1 (to find it filter by language: German) but with the lang of the course in parentheses. All the other courses that are available in different languages, have followed option 3.

I also support option 1.

**Comment by shawna-slh** [comment]
If @leticiaseixas @kakinney @bakkenb agree, I support adding a brief note in the submission form, publishing it right away so the new submitters starting today see it, and adding it to the list at https://www.w3.org/WAI/EO/wiki/EOWG_Meetings#Minor_Changes_for_Review.
per [EOWG Process for Minor Changes](https://www.w3.org/WAI/EO/wiki/EOWG_Participation_Info#Process_for_Minor_Changes)

**Comment by leticiaseixas** [comment]
From what I understand, the discussion leans toward:
- providing the course title in both languages (English in parentheses)
- all the information in the language of the course

The following text was added to the submission form ([preview](https://deploy-preview-349--wai-course-list.netlify.app/courses/submission/)):

> "You are welcome to submit a course in a language other than English. For that, please indicate the course title in its original language and, in parentheses, its corresponding name in English. You are also encouraged to fill in the information requested in this form in the language of the course."

**Comment by carlosapaduarte** [comment]
I misunderstood what was being proposed, and therefore I'm going to clarify my position. I would encourage the course title to be provided in both languages, but not anything else. Therefore, I agree with the first two sentences of the additional text, but not the final one. 

Having everything described in two languages would make it:
- confusing for the users of the list
- impossible for us, reviewers, to validate (unless we are sure that automated translators do not make mistakes)

**Comment by shawna-slh** [comment]
/me suggested wording tweaks at https://github.com/w3c/wai-course-list/pull/350/files before reading this comment.

I've validated lots and lots of [translations WAI resources](https://www.w3.org/WAI/translations/), that are substantially longer and need to be accurate translations. I have found that with machine translation I can spot potential issues, and ask the translator clarifying questions if needed. (We also get review from native speakers. Per below, I don't think that's at all needed for course submissions.)

For this course list, you only need to validate that it's a legitimate submission and the information is generally accurate. I think you can do that sufficiently with machine translation. (Yes, it is a bit more work.)

Here are some tips: https://www.w3.org/wiki/WAI/Translation_Instructions#For_people_who_do_not_know_the_translated_language

Based on the "requirements", the primary target users and tasks are looking for a course for themselves (individuals or organizations), so it **seems better to have the description and topics in their language**?

... although, given the rest of the info is in English ... 
Maybe worth getting more input from other multi-language EOWG folks and others?

**Comment by bakkenb** [comment]
I am not a representative of the multi-language community (English being my only language); I would also like to hear from those that use multiple languages for their perspective.

That being said, I would agree with Shawn's take on the rest of the information being in the language of the course...

> You may provide the description and other course information in only the language of the course, or also in English.

That way it will further confirm that in fact the course itself is not an English based course. 

To summarize: 

- Agree that language of course is first in Title
- English translation in parenthesis is second in Title
- Other information is provided in the language of the course
- If room, and if so inclined, submitter can add details in English, in parenthesis, after the details of the language of the course

**Comment by shawna-slh** [comment]
@carlosapaduarte @leticiaseixas
Looks like you might be getting a bunch in other languages soon! https://twitter.com/reinaldoferraz/status/1571880211276505088

It would be good to get that data in the language that we want now, so we don't have to go back and ask for updates later.

I propose: **asap** add the first part that I think we all agree on:
> Other Languages: If your course is in a language other than English, please put the course title in that language, and the English translation in parentheses.

Then very soon get general consensus on the second part ("You may provide the description and other course information in only the language of the course, or also in English."), so we can add that soon, too. 

_(Reminder that you need to "ping" me by mobile when you want me to publish updates, since I'm mostly off this week.)_

**Comment by shawna-slh** [comment]
sidenote: reminder that we need to wrap text in other languages with ```lang``` [H58](https://www.w3.org/WAI/WCAG22/Techniques/html/H58) e.g.,:
```<a href="https://www.incite.at/de/kurse-zertifikate/webaccessibility/">WebAccessibility - Accessible Webdesign (German: <span lang="de">Lehrgang WebAccessibility - Barrierefreies Webdesign</span>)</a>```

**Comment by leticiaseixas** [comment]
Update message with the first part that I think we all agree on: [Preview](https://deploy-preview-349--wai-course-list.netlify.app/courses/submission/)

> Other Languages: If your course is in a language other than English, please put the course title in that language, and the English translation in parentheses.

**Comment by kakinney** [comment]
I would also like to hear from people that speak languages other than English, but all in all, I agree with Brent's summary.  
> To summarize:
> 
> * Agree that course title is provided first in the language of course
> * English translation of title is second and in parenthesis
> * Other information is provided in the language of the course
> * If room, and if so inclined, submitter can add details in English, in parenthesis, after the details of the language of the course

I think if a course is in German, then the description should also be in German, and the English is secondary, since the course is not available in English.

**Comment by carlosapaduarte** [comment]
Another couple of arguments have appeared:

- As Shawn [pointed out](https://github.com/w3c/wai-course-list/issues/347#issuecomment-1251220587) text in a different language needs to be marked as such. I don't think this will be easily automated. So far, @leticiaseixas has been entering this directly in a file (see [this PR](https://github.com/w3c/wai-course-list/pull/355/files) for an example). But this is far from being efficient. Unless there is an automated way to recognise the language and mark it when generating the files, that I'm not aware (we can resort to the language of the course specified in the submission because parts of the submission can be in English, or multiple languages can be provided in the form)
- What happens when a course is provided in multiple languages? Should the provider submit two (or more) courses, each in one of the languages? We currently have one instance of a course that has the same program but two versions: one Spanish and one Italian. The team is different, so we probably can accept these are in fact two different courses (but it's such a fine line). But, what if the team was the same?
